project/AppDependencies.scala

```diff
@@ -17,12 +17,12 @@ object AppDependencies {
     "org.scalatestplus"           %% "scalatestplus-scalacheck" % "3.1.0.0-RC2",
     "org.scalatestplus"           %% "mockito-3-12"             % "3.2.10.0",
     "org.pegdown"                 %  "pegdown"                  % "1.6.0",
-    "org.jsoup"                   %  "jsoup"                    % "1.14.3",
+    "org.jsoup"                   %  "jsoup"                    % "1.15.1",
     "com.typesafe.play"           %% "play-test"                % PlayVersion.current,
     "org.mockito"                 %  "mockito-all"              % "1.10.19",
```

Comment:
This is outdated, please replace with `"org.mockito"                 %% "mockito-scala"            % "1.17.12",`. This will also require quite a few changes to tests being:

- Replace `import org.mockito.Matchers` with  `import org.mockito.ArgumentMatchers`
- Replace `import org.scalatestplus.mockito.MockitoSugar` with `import org.mockito.MockitoSugar`

`"org.scalatestplus"           %% "mockito-3-12"             % "3.2.10.0",` can also be removed, this will need this change too:
- `import org.scalatestplus.mockito.MockitoSugar` will need replacing with `import org.mockito.MockitoSugar`

The easiest way to do this is to do a find and replace in all files in your IDE with the new changes